CRM/Contribute/BAO/Contribution/Utils.php

```diff
@@ -652,4 +652,17 @@ public static function overrideDefaultCurrency($params) {
     $config->defaultCurrency = CRM_Utils_Array::value('currency', $params, $config->defaultCurrency);
   }
 
+  /**
+   * Get either the public title if set or the title of a contribution page for use in workflow message template
```

Comment:
Jenkins doesn't enforce this but there is supposed to be a full stop & empty line according to the full standard